roles/libvirt_manager/tasks/create_networks.yml

```diff
@@ -81,6 +81,36 @@
   loop_control:
     label: "{{ item.name }}"
 
+# NOTE(hjensas): Using a random string for dummy interface names because
+#                otherwise we hit the 15 char limitation on ifname.
+- name: Create dummy interfaces to ensure bridges are UP
+  vars:
+    cifmw_ci_nmstate_unmanaged_node_config: >-
+      {%- set interfaces = []                                            -%}
+      {%- for network in networks | map(attribute="name") | flatten
+            if cifmw_libvirt_manager_network_interface_types[network | regex_replace('cifmw[-_]','')]
+            | default('bridge') == 'bridge'                              -%}
+      {%- set suffix = lookup('password',
+                              '/dev/null',
+                              chars=['ascii_lowercase', 'digits'],
+                              length=8,
+                              seed=network)                              -%}
+      {%- set _ = interfaces.append(
+        {
+          'name': 'dummy-' + suffix,
```

Comment:
[Suggestion]
I think we should add a description[1] to explain why we are creating this dummy interface. Just because this is unexpected, I can imagine people wondering why this dummy interface exists when troubleshooting some other networking issue.

[1] https://nmstate.io/devel/yaml_api.html#description